tests/functional/registry/test_deployment.py

```diff
@@ -88,3 +95,14 @@ def test_experimental_deployments(
     )
     registry.endorseVault(experimental_vault, {"from": gov})
     assert registry.latestVault(token) == experimental_vault
+
+
+def test_governance(guardian, rewards, registry, create_token, create_vault, rando):
+    # Not just anyone can create a new endorsed Vault, only governance can!
+    with brownie.reverts():
+        registry.newVault(create_token(), guardian, rewards, "", "", {"from": rando})
+
+    # Not just anyone can endorse a Vault either
+    vault = create_vault()
+    with brownie.reverts():
+        registry.endorseVault(vault, {"from": rando})
```

Comment:
This seems better under `test_experimental_deployments` because it is intended that a Vault can be "promoted" from experimental to endorsed, and this is one of the authorization checks for that logic

Reply:
Ah ok. Moved.